terraform_files/baremetal/main.tf

```diff
@@ -191,3 +192,9 @@ data "libvirt_network_dns_host_template" "console" {
   ip       = var.single_node_ip
   hostname = "console-openshift-console.apps.${var.cluster_name}.${var.cluster_domain}"
 }
+
+data "libvirt_network_dns_host_template" "canary" {
+  count    = var.bootstrap_in_place && var.single_node_ip != "" ? 1 : 0
+  ip       = var.single_node_ip
+  hostname = "canary-openshift-ingress-canary.apps.${var.cluster_name}.${var.cluster_domain}"
```

Comment:
maybe always? Why do u think it's specific to single node? I would say it's specific for none platform

Reply:
we should do some cleanup, because single-node uses the baremetal template and the other none-platform flow uses a separate template (yes, it's pretty ugly)
I guess we should first decide if we're building single-node above none platform or baremetal, and then do some cleanup in our templates

Either way I cannot add it to none template without it being covered on prow/jenkins

Reply:
anyway added to the none platform template